### ISSUE TITLE
Mower State Description not in database! 266

### ISSUE THREAD

**Issue by eavanvalkenburg** [jm-73/Indego#73]
I got this message today, I have a Indego S+ 350 Connect. I think it was mowing using a smartschedule at the time.

**Comment by jm-73** [comment]
Do you have any clue what the mower did at the time for this state?

**Comment by jm-73** [comment]
I need some more information...

**Comment by eavanvalkenburg** [comment]
If I remember correctly it was mowing while in smartmowing mode

**Comment by jm-73** [comment]
All the other state messages that are in the 260-range are describing something related to docking:
```
'260'  : 'Docked',
'261'  : 'Docked',
'262'  : 'Docked - Loading map',
'263'  : 'Docked - Saving map',
```
Do you often get this state?

**Comment by jm-73** [comment]
Need information about what the mower or the app says that the mower is doing while it is reporting state 266. I can observer in logs that the state seems to occur when mowing. Please send log files to me (PM via Discord) and I will se if I could figure this out. And you could also help med by observing the messages on the Mower display and try to correlate this to this "266 Description not found" in your integration log file.

**Comment by jm-73** [comment]
Fixed in 1.6
